app/views/groups/assign_scans.html.erb

```diff
@@ -0,0 +1,287 @@
+                  </div>
+                </div>
+              </div>
+              <div id="documentPropertiesOverlay" class="container hidden">
+                <div class="dialog">
+                  <div class="row">
+                    <span data-l10n-id="document_properties_file_name">File name:</span> <p id="fileNameField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_file_size">File size:</span> <p id="fileSizeField">-</p>
+                  </div>
+                  <div class="separator"></div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_title">Title:</span> <p id="titleField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_author">Author:</span> <p id="authorField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_subject">Subject:</span> <p id="subjectField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_keywords">Keywords:</span> <p id="keywordsField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_creation_date">Creation Date:</span> <p id="creationDateField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_modification_date">Modification Date:</span> <p id="modificationDateField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_creator">Creator:</span> <p id="creatorField">-</p>
+                  </div>
+                  <div class="separator"></div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_producer">PDF Producer:</span> <p id="producerField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_version">PDF Version:</span> <p id="versionField">-</p>
+                  </div>
+                  <div class="row">
+                    <span data-l10n-id="document_properties_page_count">Page Count:</span> <p id="pageCountField">-</p>
+                  </div>
+                  <div class="buttonRow">
+                    <button id="documentPropertiesClose" class="overlayButton"><span data-l10n-id="document_properties_close">Close</span></button>
+                  </div>
+                </div>
+              </div>
+            </div>  <!-- overlayContainer -->
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div id='drag'></div>
+
+    <div id='right-pane'>
+      <div class='tab-pane flex-pane' id='mark_pane'>
```

Comment:
For the PDF viewer removing certain HTML elements actually break the it and the javascript calls won't work. Currently I have it set so that all the elements not needed are display: none